docs/src/prototypes/customScrollbar/index.tsx

```diff
@@ -0,0 +1,153 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import Scrollbars from 'react-custom-scrollbars'
+import {
+  Text,
+  Menu,
+  List,
+  Button,
+  Popup,
+  Dialog,
+  ShorthandValue,
+  MenuProps,
+} from '@stardust-ui/react'
+import { PrototypeSection, ComponentPrototype } from '../Prototypes'
+
+const submenuWithScrollbars = (shorthand: ShorthandValue<MenuProps>, height: string) => {
```

Comment:
```suggestion
const submenuWithScrollbars = (menu: MenuItemProps['menu'], height: string) => {
```

This will be more safer and maintainable :)